intranet/static/js/common.nav.js

```diff
@@ -0,0 +1,39 @@
+$(function() {
```

Comment:
This is still annoying and should be fixed, with I would imagine some
config change in Codacy.